### PR TITLE
CVP-1871 Enable more checks for ansible linting in upstream playbooks

### PR DESCRIPTION
**Pull request by LikhithaEda**
**CVP-1871**
The changes I made are getting rid of ansible-lint warnings & errors. 
Changes were made in all yml files.
I resolved errors such as : 
   - too many spaces before colon  (colons)
   - too few spaces before comment  (comments)
   - line too long (98 > 80 characters)

### CODE REVIEW

File: ansible-lint.yml [hunk 1]
```diff
@@ -24,4 +24,4 @@ skip_list:
   - '403'
   - '502'
   - '601'
-  - '602'
\ No newline at end of file
+  - '602'
```
Comment (samvarankashyap) [inline ansible-lint.yml:27]: @LikhithaEda Why is 602 added. 
which rule is it.

**Review by 14rcole — APPROVED** [review]
LGTM nice work

**Comment by samvarankashyap** [comment]
@LikhithaEda 
Please change the title of the PR to following format

CVP-1871 <title of the ticket >

**Review by LikhithaEda — COMMENTED** [review]
> @LikhithaEda
> Please change the title of the PR to following format
> 
> CVP-1871 <title of the ticket >

done

**Comment by lslebodn** [comment]
@LikhithaEda  Please rebase on top of current master

**Comment by lslebodn** [comment]
BTW @mvalarh introduced some config file as well as part of his change
https://github.com/redhat-operator-ecosystem/operator-test-playbooks/blob/master/.ansible-lint

It would be good to use just a one file

**Review by mvalarh — APPROVED** [review]

**Review by lslebodn — CHANGES_REQUESTED** [review]
https://github.com/redhat-operator-ecosystem/operator-test-playbooks/pull/157/checks?check_run_id=2892725111

The check  "Run ansible-lint and yamllint for this repo. / build" still do not pass

-1

**Comment by lslebodn** [comment]
@LikhithaEda  Could you explain why new file local.yml was added in the pull request.
We were supposed to fix linting issues. I cannot see a relation with adding new files.

Moreover, I can still se two configuration for ansible lint in different files.
* https://github.com/LikhithaEda/operator-test-playbooks/blob/local/.ansible-lint
* https://github.com/LikhithaEda/operator-test-playbooks/blob/local/ansible-lint.yml

We should use just one configuration.

**Review by lslebodn — CHANGES_REQUESTED** [review]
Some of question still has not been answered ^^

**Review by LikhithaEda — COMMENTED** [review]
> https://github.com/redhat-operator-ecosystem/operator-test-playbooks/pull/157/checks?check_run_id=2892725111
> 
> The check "Run ansible-lint and yamllint for this repo. / build" still do not pass
> 
> -1

I solved this issue in the new commit

**Comment by LikhithaEda** [comment]
> @LikhithaEda Could you explain why new file local.yml was added in the pull request.
> We were supposed to fix linting issues. I cannot see a relation with adding new files.
> 
> Moreover, I can still se two configuration for ansible lint in different files.
> 
>     * https://github.com/LikhithaEda/operator-test-playbooks/blob/local/.ansible-lint
> 
>     * https://github.com/LikhithaEda/operator-test-playbooks/blob/local/ansible-lint.yml
> 
> 
> We should use just one configuration.

I did not add the local.yml file. It was already there and just modified to fix linting errors. I think after the rebase it is showing as a new file. And the reason the check failed previously is because of a syntax error in the local.yml which I fixed this time.

**Comment by LikhithaEda** [comment]
> BTW @mvalarh introduced some config file as well as part of his change
> https://github.com/redhat-operator-ecosystem/operator-test-playbooks/blob/master/.ansible-lint
> 
> It would be good to use just a one file
I rebased after this changes were merged but I don't see this file locally for some reason. Should I add it myself?

**Comment by lslebodn** [comment]
> @lslebodn I did not add the local.yml file. It was already there and just modified to fix linting errors. I think after the rebase it is showing as a new file. And the reason the check failed previously is because of a syntax error in the local.yml which I fixed this time.

Github says you added the file.
* https://patch-diff.githubusercontent.com/raw/redhat-operator-ecosystem/operator-test-playbooks/pull/157.patch
```
diff --git a/local.yml b/local.yml
new file mode 100644
index 00000000..f93dc3cf
--- /dev/null
+++ b/local.yml
@@ -0,0 +1,112 @@
+---
+- name: Test
```

**Comment by lslebodn** [comment]
@samvarankashyap  please help @LikhithaEda  with rebase

**Review by lslebodn — CHANGES_REQUESTED** [review]
The story says:
```
Remove as much rules from the ansible-lint.yml skip_list as possible
```
and there are also list of rules which should be removed from the configuration file.
But we did not change anything in configuration.
There is just new-line added.

Fixing linting violation is nice but we need to ensure nobody else will introduce them in the future. Which is not done in current MR.

We should also remove duplicate configuration for ansible lint
* https://github.com/LikhithaEda/operator-test-playbooks/blob/local/.ansible-lint
* https://github.com/LikhithaEda/operator-test-playbooks/blob/local/ansible-lint.yml

There is not any reason to have two different files for the same purpose.

**Comment by LikhithaEda** [comment]
> The story says:
> 
> ```
> Remove as much rules from the ansible-lint.yml skip_list as possible
> ```
> 
> and there are also list of rules which should be removed from the configuration file.
> But we did not change anything in configuration.
> There is just new-line added.
> 
> Fixing linting violation is nice but we need to ensure nobody else will introduce them in the future. Which is not done in current MR.
> 
> We should also remove duplicate configuration for ansible lint
> 
>     * https://github.com/LikhithaEda/operator-test-playbooks/blob/local/.ansible-lint
> 
>     * https://github.com/LikhithaEda/operator-test-playbooks/blob/local/ansible-lint.yml
> 
> 
> There is not any reason to have two different files for the same purpose.

The two files: **.ansible-lint** and **ansible-lint.yml** have different contents.

**Comment by wheelerlaw** [comment]
@LikhithaEda and I merged the `.ansible-lint` and `ansible-lint.yml` file into a single `.ansible-lint` file.

**Comment by wheelerlaw** [comment]
> /retest

Lihkitha will need to rebase on top of the current master in order for the checks to pass.

File: .ansible-lint [hunk 2]
```diff
@@ -17,4 +16,14 @@ skip_list:
   - command-instead-of-shell  # Use shell only when shell functionality is required
   # - yaml  # Violations reported by yamllint
   - line-length
-  - comments
+  - '102'
```
Comment (14rcole) [inline .ansible-lint:19]: Can you add comments to these lines saying what each of the errors is

**Comment by 14rcole** [comment]
LGTM

**Comment by wheelerlaw** [comment]
lgtm. let's get this merged in for now and we can make more changes in the future.

**Review by wheelerlaw — APPROVED** [review]

**Pull request merged**